Makefile

```diff
@@ -210,3 +210,10 @@ update-chart-dep: dep-update ## Updates the dependency version in the Chart.yaml
 .PHONY: prepare-release
 prepare-release: ## Prepares for a new release of the helm chart. Optionally specify CHART_VERSION and APP_VERSION.
 	ci_scripts/prepare-release.sh CREATE_BRANCH=${CREATE_BRANCH} CHART_VERSION=${CHART_VERSION} APP_VERSION=${APP_VERSION}
+
+.PHONY: tidy
+tidy:
```

Comment:
Would a target like `tidyall` be more clear here, just to make it obvious that this is for all modules in the repo?